airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,15 @@
 Changelog
 ---------
 
+8.0.0
+......
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+* ``Remove deprecated 'delegate_to' param from transfers that interact with Google Cloud (#30748)``
```

Comment:
This line will be added automatically by the release manager during release.
What we want to describe via the PR is information about that the commit message itself does not cover.
For example: what operators involve? what action users needs to take.. basically additional information that may help user to bridge the breaking changes faster

Reply:
Done